mtags/src/main/scala-3/scala/meta/internal/pc/completions/NamedArgCompletions.scala

```diff
@@ -0,0 +1,113 @@
+package scala.meta.internal.pc.completions
+
+import scala.meta.internal.mtags.MtagsEnrichments.*
+
+import dotty.tools.dotc.ast.tpd.Apply
+import dotty.tools.dotc.ast.tpd.Block
+import dotty.tools.dotc.ast.tpd.Ident
+import dotty.tools.dotc.ast.tpd.Literal
+import dotty.tools.dotc.ast.tpd.NamedArg
+import dotty.tools.dotc.ast.tpd.Select
+import dotty.tools.dotc.ast.tpd.Template
+import dotty.tools.dotc.ast.tpd.Tree
+import dotty.tools.dotc.core.Constants.Constant
+import dotty.tools.dotc.core.Contexts.Context
+import dotty.tools.dotc.core.Flags
+import dotty.tools.dotc.core.NameKinds.DefaultGetterName
+import dotty.tools.dotc.core.Names.Name
+import dotty.tools.dotc.core.Symbols.Symbol
+import dotty.tools.dotc.interactive.Completion
+import dotty.tools.dotc.util.SourcePosition
+
+object NamedArgCompletions:
+
+  def contribute(
```

Comment:
I named this method `contribute` just because I followed what Scala2 version of `CompletionPosition` abstract class does: https://github.com/tanishiking/metals/blob/0868c44b5bec2ff69e3cda2e51fa9ad8d8b779ba/mtags/src/main/scala-2/scala/meta/internal/pc/completions/Completions.scala#L348-L388 

As Scala3 version of completions don't have a common base class, we can rename `contribute` to `complete` or something that make more sense than `contribute`.
(It's OK to stay `contribute` in this PR!)

Reply:
Yep, I think it fine leave it as `contribute` to keep the similarity with scala2 impl.